src/Shell.php

```diff
@@ -1224,7 +1205,7 @@ public function getLastExecSuccess(): bool
     public function formatException(\Exception $e): string
     {
         if ($e instanceof BreakException) {
-            return \sprintf('  <fg=white;bg=blue;options=bold> INFO </> %s.', \rtrim($e->getRawMessage(), '.'));
+            return \sprintf('  <info> INFO </info> %s.', \rtrim($e->getRawMessage(), '.'));
```

Comment:
Is it possible to ensure padding inside `<info>` in the style definition, so whitespaces won't be required explicitly here? I mean `<info>INFO</info>`. Or maybe it works already and those space are superfluous here?

Reply:
I am not sure how to accomplish this.

Reply:
Me neither 😅 Probably [`SymfonyStyle`](https://github.com/symfony/symfony/blob/6.1/src/Symfony/Component/Console/Style/SymfonyStyle.php) could help, I'm pretty sure there is something similar in their commands' output. But it's up to you - from my PoV it can stay like this, without changes 🙂

Reply:
I don't think so. IIRC it's just for colors.